.gitignore

```diff
@@ -35,3 +35,7 @@ node_modules
 /cjs
 /umd
 /es
+
```

Comment:
I don't think these additions are required. As far as I understand, everything is built into either `cjs`, `umd` or `es`.